src/app/series/directives/series-plan.component.html

```diff
@@ -8,6 +8,19 @@ <h1>Plan Episodes</h1>
   </p>
 </header>
 
+<section>
+  <h2>What time of day do you plan to publish your episodes?</h2>
```

Comment:
`<feedback-not-demand-for-change>`

I don't love that this is the first section in the episode planner, or the description text below.  But I have no better suggestions (page is looking a bit crowded/busy in general, and words are hard).  So I'd defer to @brandonhundt on layout.

![image](https://user-images.githubusercontent.com/1410587/67419454-f5857680-f589-11e9-8ce8-c4f87c311ee4.png)